src/test/resources/features/acceptance/address-lookup.feature

```diff
@@ -87,3 +87,8 @@ Feature: Select address
     Then I am shown the select address page
     And I am informed that there's a problem with the address lookup
     And I am shown a link to enter my address manually
+
+  Scenario: I am shown an error response when I click continue without selecting an address
```

Comment:
Maybe doesn't need the word `response`:
```
I am shown an error when I click continue without selecting an address
```

Reply:
All error scenarios in this feature include 'I am shown an error response'.